platform-cloud/docs/monitoring/run-details.mdx

```diff
@@ -158,6 +159,22 @@ Select a task in the task table to open the **Task details** dialog. The dialog
 
   If the pipeline work directory is in cloud storage, this tab shows a [Data Explorer](../data/data-explorer.mdx) view of the task's work directory location with the files associated with the task.
 
+- **Container**
+
+  The **Container** tab shows the container details used to run the task, including links to Wave reports, if Wave is used.
```

Comment:
```suggestion
  The **Container** tab shows the container details used to run the task.
```

Container details are reported irrespective of Wave. When using Wave more details are available tho